packages/docs/src/pages/index.md

```diff
@@ -3,6 +3,13 @@ title: Introduction
 weight: -100
 ---
 
+<div class="ds-c-alert">
+  <div class="ds-c-alert__body">
+    <h3 class="ds-c-alert__heading">Tell us how the system is working for you</h3>
```

Comment:
Can we make this tag an `<h2>`? I tried it out with dev tools and there was no visual change, so I'm all for boosting semantics unless it causes a problem.